### PR TITLE
Add sanctuary

### PR DESCRIPTION
**Pull request by gillchristian**
This PR's adds [sanctuary](https://sanctuary.js.org) & [fluture](https://github.com/fluture-js/Fluture) to replace `data.task`, `data.either` and `data.maybe`.

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const fs = require('fs')
-const Task = require('data.task')
+const Task = require('fluture')
```
Comment (fvictorio) [inline lib/utils.js]: Can we do `const Future = require('fluture')` instead? Or do you like `Task` better?

File: package.json [hunk 2]
```diff
@@ -29,14 +29,14 @@
...
-    "data.task": "^3.1.1",
-    "immutable": "^3.8.1",
-    "immutable-ext": "^1.0.8",
+    "fluture": "^7.1.3",
+    "fluture-sanctuary-types": "^1.3.0",
     "meow": "^3.7.0",
     "ora": "^1.2.0",
     "pluralize": "^4.0.0",
-    "ramda": "^0.23.0"
+    "ramda": "^0.23.0",
```
Comment (fvictorio) [inline package.json]: Are we still using Ramda somewhere?

Comment (gillchristian) [reply package.json]: Yes, we could remove Ramda (`R`) in favor of Sanctuary (`S`), but `R` is a bit more flexible (i.e.
 [`S.compse`](https://sanctuary.js.org/#compose) vs [`R.compose`](http://ramdajs.com/docs/#compose) ).

For a rule of thumb I would favor `S` as much as possible, but when it makes things more complicated I'd use `R`, wdyt ? Still 99% of the cases we could use `S` so maybe we just have to stick to it.

File: lib/utils.js (hunk 1 above)
Comment (gillchristian) [reply lib/utils.js]: Sure, let's do it.

**Comment by gillchristian** [comment]
> I didn't check all the code, I will have faith in your FP prowess.

:joy:

**Pull request merged**